Lib/elements/text_area.py

```diff
@@ -0,0 +1,32 @@
+"""Class for handling Text area elements."""
+from base_element import BaseElement
+
+
+class TextArea(BaseElement):
+    def __init__(self, driver, element_tuple, timeout=10):
```

Comment:
similar concern as for previous element class, if you update previous one please do not forget to update other in the similar way